### PR TITLE
[INTEL MKL]: ADD Tensorflow+MKL Singularity definition files

### PR DESCRIPTION
**Pull request by karthikvadla**
This PR adds singularity definition files for intel optimized tensorflow.

Definition files are created based of existing docker binaries from docker hub. This helps for easier maintenance

### CODE REVIEW

**Comment by ashahba** [comment]
@karthikvadla Do we want recipes for `Python3` images as well?
If yes then we need to add files specific to below images:
```
intelaipg/latest-mkl-py3
intelaipg/latest-devel-mkl-py3
```

**Comment by karthikvadla** [comment]
@penpornk can you please review this PR when you get time.

**Comment by penpornk** [comment]
Thank you for your PR and sorry for my delay! The scripts' content look fine. I just have to make sure if this is the right place to put them, and about the license term. Looping in @thirupalanisamy and @tatianashp for more comments.

**Comment by karthikvadla** [comment]
Sure thank you. @ashahba can help with license things from our end.

**Comment by karthikvadla** [comment]
> @karthikvadla Do we want recipes for `Python3` images as well?
> If yes then we need to add files specific to below images:
> 
> ```
> intelaipg/latest-mkl-py3
> intelaipg/latest-devel-mkl-py3
> ```

If you think that really helps others we can do it. or based on requirement we can open separate PR's.

**Comment by thirupalanisamy** [comment]
> Thank you for your PR and sorry for my delay! The scripts' contents look fine. I just have to make sure if this is the right place to put them, and about the license term. Looping in @thirupalanisamy and @tatianashp for more comments.

Apache 2.0 license is already part of our OSS approved list, we are good there.

**Comment by penpornk** [comment]
Note on copyright for future new files: It should be

**Comment by penpornk** [comment]
Ahh sorry. I clicked the wrong button! Will try to reopen it.

**Review by penpornk — COMMENTED** [review]
According to @martinwicke,
The scripts should live with the docker files you reference, rather than here (TensorFlow github). I've requested a review from him in case you have further comments.

File: tensorflow/tools/singularity/Singularty-devel-mkl [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2018 Intel Corporation
```
[the remark this answers is not in the document]
Comment (karthikvadla) [reply tensorflow/tools/singularity/Singularty-devel-mkl]: yeah sure @penpornk . Will update that.

**Comment by karthikvadla** [comment]
@penpornk should i create `singularity` sub-folder at below structure and add files?
https://github.com/tensorflow/tensorflow/tree/master/tensorflow/tools/docker .

I can add section for singularity too in readme, if you are okay with that.

File: tensorflow/tools/singularity/Singularty-devel-mkl (hunk 1 above)
[the remark this answers is not in the document]
Comment (karthikvadla) [reply tensorflow/tools/singularity/Singularty-devel-mkl]: and where can i find the authors list, can you please point me to that.

**Comment by karthikvadla** [comment]
> I don't think these files should live in tensorflow/tensorflow. The underlying dockerfiles are inintelaipg/intel-optimized-tensorflow? Then these files should be there as well.
> 
> That also makes the discussion about licenses and directories moot.

oh this is https://github.com/tensorflow/tensorflow/blob/c1738f64a7aa900bb7de422c36ebd93686c46fee/tensorflow/tools/singularity/Singularty-devel-mkl#L22 reference to binary on docker hub https://hub.docker.com/r/intelaipg/intel-optimized-tensorflow/tags/ , not to any dockerfiles.

**Pull request closed without merging**